src/sentry/static/sentry/app/icons/iconActivity.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {IconProps} from 'app/types/iconProps';
+import theme from 'app/utils/theme';
+
+export const IconActivity: React.FC<IconProps> = ({
+  color: providedColor = 'currentColor',
+  size: providedSize = 'sm',
+  ...props
+}: IconProps) => {
+  const color = providedColor;
+  const size =
+    typeof providedSize === 'string' ? theme.iconSizes[providedSize] : providedSize;
```

Comment:
We may want to change this up, we could have strings that are valid sizes, right?

What about if we check that `providedSize` is in `theme.iconSizes` otherwise use `providedSize`? e.g. `const size = theme.iconSizes[providedSize] ?? providedSize`

Reply:
Sure